src/components/Display.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable react/require-default-props */
+/* eslint-disable react/default-props-match-prop-types */
```

Comment:
[Optional]

These rules are unnecessary, let's remove it and the `isRequired,` in the `result` propTypes definition. According to the [rule](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/default-props-match-prop-types.md), propTypes shouldn't be required and have default props at the same time.

```diff
Display.propTypes = {
-  result: PropTypes.string.isRequired,
+  result: PropTypes.string,
};
```